static/app/components/acl/access.tsx

```diff
@@ -60,16 +60,10 @@ function Access({
   organization,
 }: Props) {
   const config = useLegacyStore(ConfigStore);
+  team = team ?? undefined;
+  project = project ?? undefined;
```

Comment:
do you need to do this if null/undefined evaluate to false?

Reply:
and `hasEveryAccess` is just a bunch of boolean checks?

Reply:
> do you need to do this if null/undefined evaluate to false?

@cathteng The props, `team` and `project` may be `null`.  Note that `hasEveryAccess` doesn't access `null` values. That's why I'm coercing them to be `undefined` beforehand.